gomock/controller.go

```diff
@@ -182,6 +183,11 @@ func (ctrl *Controller) Finish() {
 	ctrl.mu.Lock()
 	defer ctrl.mu.Unlock()
 
+	if ctrl.finished {
+		ctrl.t.Fatalf("Controller.Finish has to be called exactly once")
```

Comment:
I'd like the error message to tell the user what went wrong in addition to explaining why it was wrong. For example:
"Controller.Finish was called more than once. It has to be called exactly once."

Reply:
Done.